src/config/util.ts

```diff
@@ -177,37 +180,15 @@ export const getUniswapPrice = async (contract: string): Promise<TokenPrice> =>
   };
 };
 
-export const getSushiswapPair = async (token: string, block?: number) => {
-  const query = `
-    {
-      pair(id: "${token.toLowerCase()}"${block ? `, block: {number: ${block}}` : ''}) {
-				id
-        reserve0
-        reserve1
-        token0 {
-          id
-					symbol
-					name
-					decimals
-        }
-        token1 {
-          id
-					symbol
-					name
-					decimals
-        }
-        totalSupply
-      }
-    }
-  `;
-  return await fetch(SUSHISWAP_URL, {
-    method: 'POST',
-    body: JSON.stringify({ query }),
-  }).then((response) => response.json());
-};
-
 export const getSushiswapPrice = async (contract: string): Promise<TokenPrice> => {
```

Comment:
what do you think about moving this pricing helper to `PricesService` similar to `getContractPrice` and `getTokenPrice`

Reply:
Discussed offline, will refactor to a separate util